src/vis/bar/BarVis.tsx

```diff
@@ -100,6 +115,9 @@ export function BarVis({ config, optionsConfig, extensions, columns, setConfig,
         itemclick: false,
         itemdoubleclick: false,
       },
+      font: {
+        family: 'Roboto',
```

Comment:
```suggestion
        family: 'Roboto, sans-serif',
```

I think we should change it to the same value that we have in the [_base.scss file](https://github.com/datavisyn/tdp_core/blob/f7c47fedd5dc41ab3d538b6656c42e3775eb3227/src/scss/base/_base.scss#L3). I'm not sure if it works here and if the syntax is correct. Please adapt for the other visualizations accordingly.

Reply:
Done, seems to work everywhere